suite/e2e/tests/onboarding/t2t1/t2t1-entropy-check-failure.test.ts

```diff
@@ -0,0 +1,94 @@
+                    category: TestCategory.Onboarding,
+                    priority: TestPriority.High,
+                }),
+            },
+            async ({ page, onboardingPage, devicePrompt, trezorUserEnvLink }) => {
+                await page.ensureStoreOnDesktop();
+
+                await page.evaluate(
+                    action => window.store.dispatch(action),
+                    deviceActions.setSimulatedEntropyCheckFail({
+                        success: false,
+                        payload: { code: 'Failure_EntropyCheck', error: 'SIMULATED ERROR' },
+                    }),
+                );
+
+                await test.step('Start creating wallet', async () => {
+                    await onboardingPage.firmware.continueThroughFirmware();
+                    await onboardingPage.createWalletButton.click();
+                    await onboardingPage.selectSeedType('12-words');
+                    await devicePrompt.confirmOnDevicePromptIsShown();
+                    await trezorUserEnvLink.pressYes();
+                });
+
+                await test.step('Land on entropy check failure', async () => {
+                    await expect(onboardingPage.deviceCompromisedModal).toBeVisible();
+                });
+            },
+        );
+
+        test(
+            'Transport error (device disconnected during entropy check)',
+            {
+                annotation: createTestAnnotation({
+                    testCase:
+                        'Verify that the proper modal is shown when entropy check gets interrupted by transport error during wallet creation.',
+                    category: TestCategory.Onboarding,
+                    priority: TestPriority.High,
+                }),
+            },
+            async ({ page, onboardingPage, devicePrompt, trezorUserEnvLink }) => {
+                await page.ensureStoreOnDesktop();
+                await page.evaluate(
+                    action => window.store.dispatch(action),
+                    deviceActions.setSimulatedEntropyCheckFail({
+                        success: false,
+                        payload: {
+                            code: 'Failure_EntropyCheck',
+                            error: 'device disconnected during action',
+                        },
+                    }),
+                );
+
+                await test.step('Start creating wallet', async () => {
+                    await onboardingPage.firmware.continueThroughFirmware();
+                    await onboardingPage.createWalletButton.click();
+                    await onboardingPage.selectSeedType('12-words');
+                    await devicePrompt.confirmOnDevicePromptIsShown();
+                    await trezorUserEnvLink.pressYes();
+                });
+
+                await test.step('Display error toast, but stay on the same screen (no device compromised)', async () => {
+                    await expect(page.getByTestId('@toast/error')).toBeVisible();
```

Comment:
text assert improves value and also checks visibility

```suggestion
                    await expect(page.getByTestId('@toast/error')).toHaveText('device disconnected during action);
```

Reply:
Done :heavy_check_mark: